tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -3,7 +3,11 @@ const { Given, Then, When } = require('cucumber');
 
 When('the user browses to the files page',
 	() => {
-		const filesPage = client.page.filesPage();
+        const filesPage = client
```

Comment:
@paurakhsharma this is something where I need your help - this code block is necessary to switch back to the phoenix tab .... this is for sure the wrong location to do this .... any idea where to put this? THX

Reply:
@DeepDiver1975 I am unable to see this myself because oauth2 is not working in mine. We have never done tab switching previously so, I wouldn't know for sure. @individual-it What do you think? This looks fine for me until we find a better way? @DeepDiver1975 I think it would be better to change the scenario step definition from `the user browses to the files page` to something like `the user switches back to the phoenix page` or something like that. @phil-davis what's your thought on it?